docs/release/2022q2.md

```diff
@@ -0,0 +1,155 @@
+## 2022-07-01: 2022q2 release
+
+This release covers everything from 2022-04-01 and has been tested with avr-gcc
+v11.2.0 from upstream and arm-none-eabi-gcc 10.3-2021.10 from Arm.
+
+Breaking changes:
+
+- None.
+
+Features:
+
+- Custom OpenOCD binary path.
+- IRQ handlers for RP2040.
+- Expose transmission complete flag on STM32 UART.
+- Half transfer handler for STM32 DMA.
+- Timer `SlaveModeTrigger::Internal0`/`Internal3` for STM32F4.
+- UI Button group refactoring.
+- MovingAverage filter refactoring.
+- Support for std::chrono in STM32 timer API.
+
+Integrated Projects:
+
+- ETL upgraded to v20.29.3.
+- CrashCatcher updated to latest version.
+
+Fixes:
+
+- STM32H7 RCC clock output.
+- CAN system clock frequencies for Nucleo-F446RE/Nucleo-F446ZE boards.
+- Cortex-M ITM when `buffer.tx` option not set.
+
+New development boards:
+
+- Support for revision B-03 of the DISCO-F469NI development board as
+  `modm:disco-f469ni:b-02`.
+
+New device drivers:
+
+- MAX31855 thermocouple-to-digital converter
+- ADS101X ADC
+- MS5611 pressure sensor
+
+Known bugs:
+
+- Cannot set breakpoints on specific lines in debug profile. See #777.
+- C++20 is not fully implemented by GCC10 yet, however modm does not use all
+  features yet anyways. See #326.
+- STM32F7: D-Cache not enabled by default. See #485.
+- `lbuild build` and `lbuild clean` do not remove all previously generated files
+  when the configuration changes. See #285.
+- Generating modm on Windows creates paths with `\` that are not compatible with
+  Unix. See #310.
+- `arm-none-eabi-gdb` TUI and GDBGUI interfaces are not supported on Windows.
+  See #591.
+
+Many thanks to all our contributors.
+A special thank you to our first timers 🎉:
+
+- Christopher Durand (@chris-durand)
+- Jonas Kazem Andersen (@JKazem) 🎉
+- Niklas Hauser (@salkinium)
+- Raphael Lehmann (@rleh)
+- Rasmus Kleist (@rasmuskleist)
+- Sarah Vilete (@sarahvilete) 🎉
+- Sascha Schade (@strongly-typed)
+- Sebastian Birke (@se-bi)
+- Sebastian Tibor Bakonyvari (@twast92) 🎉
+- Thomas Rush (@tarush53) 🎉
+- Thomas Sommer (@TomSaw)
+- Vivien Henry (@lukh)
+- cocasema 🎉
```

Comment:
```suggestion
-  Nikolay Semenov (@cocasema) 🎉
```
(from the copyright header) Add it to [the mapping here](https://github.com/modm-io/modm/blob/develop/tools/scripts/authors.py#L81).